src/MOI/MOI_callbacks.jl

```diff
@@ -81,7 +87,8 @@ function default_moi_callback(model::Optimizer)
             # only allow one user cut solution per LP optimal node
             # limiting two calls to guarantee th user has a chance to add
             # a cut. if the user cut is loose the problem will be resolved anyway.
-            if Xpress.getintattrib(cb_data.model, Xpress.Lib.XPRS_CALLBACKCOUNT_OPTNODE) > 2
+            if Xpress.getintattrib(cb_data.model, Lib.XPRS_CALLBACKCOUNT_OPTNODE)> 2
+                #println("Attribute ",Lib.XPRS_CALLBACKCOUNT_OPTNODE," value recovered : ",Lib.XPRSgetintattrib(cb_data.model, Lib.XPRS_CALLBACKCOUNT_OPTNODE, _)::Int)
```

Comment:
same thing, kind of weird, there are a lot in the code

Reply:
Same as above